<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SetTitle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetSheetName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SetSheetName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSheetName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddSheet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AddSheet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteSheet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DeleteSheet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddCell">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AddCell</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
